### PR TITLE
OPT - Removed root user from Dockerfile. Defaults to https.

### PR DESCRIPTION
**Pull request by simonracz**

### CODE REVIEW

File: frontend/etcdbrowser.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ app.controller('NodeCtrl', ['$scope','$http','$cookies', function($scope,$http,$
   if($cookies.urlPrefix){
     $scope.urlPrefix = $cookies.urlPrefix;
   } else {
-    $scope.urlPrefix = "http://" + document.location.host;
+    $scope.urlPrefix = "https://" + document.location.host;
```
Comment (streeck) [inline frontend/etcdbrowser.js:20]: 🔝

**Review by streeck — APPROVED** [review]
LGTM

File: Dockerfile [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
 RUN mkdir /app
 ADD . /app/
 
+RUN groupadd -g 1024 ebrowser && \
+  useradd -u 1024 -g ebrowser ebrowser && \
+  chown -R ebrowser:ebrowser /app
+
 WORKDIR /app
-EXPOSE 8000
+EXPOSE 8080
```
Comment (jeduden) [inline Dockerfile:15]: what was your motivation to change the port ?

Comment (simonracz) [reply Dockerfile:15]: I don't remember (this is an old diff). The port change was surely not necessary.

**Pull request merged**